packages/diagrams/frontend/sirius-components-diagrams-reactflow/src/renderer/DiagramRenderer.tsx

```diff
@@ -11,9 +11,9 @@
  *     Obeo - initial API and implementation
  *******************************************************************************/
 
-import { Selection, SelectionEntry } from '@eclipse-sirius/sirius-components-core';
+import { Selection, SelectionEntry, useMultiToast } from '@eclipse-sirius/sirius-components-core';
```

Comment:
In a similar fashion as my comment on hide, fade, you could prevent the addition of additional complexity in DiagramRender using an API like this:

```
import { useReconnectEdge } from './reconnect-edge/useReconnectEdge';

// ...

const { reconnectEdge } = useReconnectEdge();
```